OpenTabletDriver.Desktop/Interop/Input/MacOSVirtualMouse.cs

```diff
@@ -1,28 +1,57 @@
 using System;
+using System.Diagnostics;
+using System.Numerics;
 using OpenTabletDriver.Native.MacOS;
 using OpenTabletDriver.Native.MacOS.Input;
+using OpenTabletDriver.Native.OSX.Input;
 using OpenTabletDriver.Platform.Pointer;
 
 namespace OpenTabletDriver.Desktop.Interop.Input
 {
     using static MacOS;
 
-    public abstract class MacOSVirtualMouse : IMouseButtonHandler, ISynchronousPointer
+    public abstract class MacOSVirtualMouse : IMouseButtonHandler, ISynchronousPointer, ITiltHandler, IEraserHandler
     {
+        private const int _DoubleClickMoveTolerance = 8;
+        private const int _ProximityExpiresDurationInMs = 200;
```

Comment:
According to Rider, our convention for private consts is PascalCasing without the underscore in front, ie:
```suggestion
        private const int DoubleClickMoveTolerance = 8;
        private const int ProximityExpiresDurationInMs = 200;
```
(the remainder of the code should be adjusted as appropriate, don't just apply this suggestion without renaming other occurences)